### PR TITLE
Outscale storage providers

### PR DESCRIPTION
**Pull request by jmunhoz**
Add Outscale storage providers (Ceph RGW S3)

### CODE REVIEW

**Comment by tonybaloney** [comment]
Hi @jmunhoz do they all need to be separate drivers? We are trying to make sure that there are single drivers with a `region=x` constructor.

**Comment by jmunhoz** [comment]
@tonybaloney thanks for reviewing. As suggested I pushed a single driver with region support in the constructor.

File: libcloud/storage/drivers/s3.py [hunk 1]
```diff
@@ -1004,3 +1013,21 @@ class S3SAEastStorageDriver(S3StorageDriver):
...
     ex_location_name = 'sa-east-1'
+
+
+class S3RGWOutscaleConnection(S3Connection):
+    pass
+
+
+class S3RGWOutscaleStorageDriver(S3StorageDriver):
+
+    def __init__(self, *args, **kwargs):
```
Comment (Kami) [inline libcloud/storage/drivers/s3.py]: Please explicitly declare all the arguments where possible instead of using `*args` and `**kwargs`.

**Comment by jmunhoz** [comment]
Hi @Kami, the new patch declares the arguments explicitly.

**Comment by tonybaloney** [comment]
Thanks @jmunhoz. the changes look great, merging

**Pull request closed without merging**